test/run-unit-tests.js

```diff
@@ -64,14 +64,52 @@ async function runJest(
   }
 }
 
+/**
+ * Run webpack tests. Webpack tests do not yet support
+ * parallelism / test-splitting.
+ *
+ * @param {boolean} coverage - Use nyc to collect coverage
+ */
+async function runWebpack({ coverage }) {
+  const options = ['tsx', '--test', './development/webpack/test/*.test.ts'];
+  // If coverage is true, then we need to run nyc as the first command
+  // and tsx after, so we use unshift to add two options to the beginning
+  // of the options array.
+  if (coverage) {
+    options.unshift('nyc', '--reporter=json', 'yarn');
+  }
+  await runInShell('yarn', options);
+  if (coverage) {
+    // Once done we rename the coverage file so that it is unique among test
+    // runners
+    await runCommand('mv', [
+      './coverage/coverage-final.json',
+      `./coverage/coverage-final-webpack.json`,
+    ]);
+  }
+}
+
 async function start() {
```

Comment:
This one is very unfortunate. I deleted this file in https://github.com/MetaMask/metamask-extension/pull/25726/files, as we needed to convert the unit tests from Circle.ci to GitHub Actions. The conversion meant we also needed to rework parallelism (as parallelism works very differently in Circle.ci and GitHub actions), which meant we could not use the `run-unit-tests.js` file anymore, as it contained Circle.ci specific stuff.

Reply:
no problem. i'll work on porting this later this week after I get caught up

Reply:
ported!